google-scripts/process-form-submission/main.ts

```diff
@@ -18,17 +18,19 @@ export function onFormSubmit(
       `${REFERRALS_SHEET_NAME}`
     );
 
-    const currentUniqueId = setUniqueIdOnSubmission(
+    const currentUniqueSubmissionDetails = setUniqueIdOnSubmission(
       referralsSheet,
       FORM_SUBMISSION_ID_COLUMN_POSITION,
       event
     );
 
-    // Update the form submission object to contain its unique ID
-    formData.FormSubmissionId = [currentUniqueId.toString()];
-
+    const uniqueId = currentUniqueSubmissionDetails.id;
+    const formRow = currentUniqueSubmissionDetails.row;
```

Comment:
Maybe we could rename this to `RowPosition`

Reply:
Done in cf5cccc